nixos/modules/services/misc/gitlab.nix

```diff
@@ -105,22 +119,22 @@ let
     GITLAB_UPLOADS_PATH = "${cfg.statePath}/uploads";
     GITLAB_LOG_PATH = "${cfg.statePath}/log";
     GITLAB_SHELL_PATH = "${cfg.packages.gitlab-shell}";
-    GITLAB_SHELL_CONFIG_PATH = "${cfg.statePath}/shell/config.yml";
+    GITLAB_SHELL_CONFIG_PATH = "${cfg.statePath}/home/config.yml";
```

Comment:
Gitlab shell now has a component written in go, and is at the $HOME by default. It's easier to just move everything in the HOME.